### ISSUE TITLE
Hi , small request

### ISSUE THREAD

**Issue by bolbolfolfol** [markusfisch/Sheeptris#1]
Hi
can you make it for android (apk)
and thank you ^....^

**Comment by markusfisch** [comment]
Yes, in theory, but Android is a completely different framework, so this would mean to port it, which takes a little time, of course 😉

**Comment by bolbolfolfol** [comment]
> Yes, in theory, but Android is a completely different framework, so this would mean to port it, which takes a little time, of course 😉

Oh, i see
if you have sometime please , if not it's ok
thank you and have a nice day ^.....^ !
